substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/JfrEvents.java

```diff
@@ -56,11 +83,19 @@ private static long getEventTypeId(String name) {
                     return type.getId();
                 }
             }
-            return 0;
+
+            String exceptionMessage = "Event " + name + " is not found! Take a look at 'metadata.xml' to see all available events.";
```

Comment:
I would rephrase this: `Event is not found!` then the suggestion which should be phrased with `most similar` and then reference to metadata.xml.

Reply:
Done.